themes/src/main/resources/theme/keycloak.v2/login/field.ftl

```diff
@@ -1,33 +1,32 @@
 <#macro group name label error="" required=false>
 
 <div class="${properties.kcFormGroupClass}">
-  <div class="${properties.kcFormGroupLabelClass}">
-    <label for="${name}" class="${properties.kcFormGroupLabelClass}">
+    <div class="${properties.kcFormGroupLabelClass}">
+        <label for="${name}" class="${properties.kcFormGroupLabelClass}">
         <span class="${properties.kcFormGroupLabelTextClass}">
-          ${label}
+            ${label}
         </span>
-        <#if required>
-          <span class="${properties.kcInputRequiredClass}" aria-hidden="true">&#42;</span>
-        </#if>
-    </label>
-  </div>
+            <#if required>
+                <span class="${properties.kcInputRequiredClass}" aria-hidden="true">&#42;</span>
+            </#if>
+        </label>
+    </div>
 
-  <#nested>
+    <#nested>
 
-  <div id="input-error-client-${name}"></div>
-  <#if error?has_content>
-    <div class="${properties.kcFormHelperTextClass}" aria-live="polite">
-      <div class="${properties.kcInputHelperTextClass}">
-        <div
-          class="${properties.kcInputHelperTextItemClass} ${properties.kcError}"
-          id="input-error-${name}">
-          <span class="${properties.kcInputErrorMessageClass}">
-              ${error}
-          </span>
-        </div>
-      </div>
+    <div id="input-error-${name}">
+        <#if error?has_content>
+            <div class="${properties.kcFormHelperTextClass}" aria-live="polite">
+                <div class="${properties.kcInputHelperTextClass}">
+                    <div class="${properties.kcInputHelperTextItemClass} ${properties.kcError}">
+                        <span class="${properties.kcInputErrorMessageClass}">
+                            ${error}
+                        </span>
+                    </div>
+                </div>
+            </div>
+        </#if>
```

Comment:
I hope the indent changes are okay. They are a mess in the entire `field.ftl` file. afaik they're supposed to be 4.

Reply:
This is correct. For those reviewing they can enable the 'hide whitespace' option in the review tab to make things a bit more legible.